frontend/app/controllers/Event.scala

```diff
@@ -143,21 +143,19 @@ trait Event extends Controller with ActivityTracking {
   }
 
   def list = CachedAction { implicit request =>
-    val pageInfo = PageInfo(
-      CopyConfig.copyTitleEvents,
-      request.path,
-      Some(CopyConfig.copyDescriptionEvents)
-    )
-    val pastEvents = (guLiveEvents.getEventsArchive ++ localEvents.getEventsArchive).headOption
-      .map(chronologicalSort(_).reverse)
     Ok(views.html.event.guardianLive(
       EventPortfolio(
         guLiveEvents.getFeaturedEvents,
         chronologicalSort(guLiveEvents.getEvents ++ localEvents.getEvents),
-        pastEvents,
+        chronologicalSort(guLiveEvents.getEventsArchive.toList.flatten ++ localEvents.getEventsArchive.toList.flatten).reverse,
```

Comment:
Forget about my comment on `.reverse`, I hadn't noticed that is called on the output of `chronologicialSort`